### PR TITLE
Set up Student Page for new Sign Up flow

### PR DESCRIPTION
**Pull request by TurnerRiley**
Sets up the 'Finish Signing up' page for students. The page currently stores all inputs in `sessionStorage`.

### New finish sign-up page for students
With the parent checkbox unchecked:
fixed components

With the parent checkbox checked:
withParent

### Demo of using fields and seeing them in `sessionStorage`
https://github.com/user-attachments/assets/495209c9-0e3b-487a-bb2f-2c63564feae8

### Page scales with screen size
student shrinks with screen size

### Right-to-left view
RTL student

## Links
Jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?assignee=60d62161f65054006980bd71&selectedIssue=ACQ-1989)
Spec: [item 7.1](https://docs.google.com/document/d/1-Bxt7orcLUsc6sHR8dKYj3XmfMNs3pG7AK8j0xeVIG8/edit#heading=h.yl293hqzpe0s)
Figma: [here](https://www.figma.com/design/88sbry7xnl1wcyrOgbgFtn/Sign-Up-Flow?node-id=2911-1878&t=PQyXNTpdIiTfmxhx-0)

## Testing story
Local testing and adding unit tests.

### CODE REVIEW

File: apps/src/signUpFlow/FinishStudentAccount.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+  return (
+    <div className={style.finishAccountContainer}>
+      <div className={style.headerTextContainer}>
+        <Heading2>{locale.finish_creating_student_account()}</Heading2>
+        <BodyTwoText>{locale.tailor_experience()}</BodyTwoText>
+      </div>
+      <div className={style.inputContainer}>
+        <div className={style.parentInfoContainer}>
+          <span className={style.parentCheckboxContainer}>
```
Comment (kelbyhawn) [inline apps/src/signUpFlow/FinishStudentAccount.tsx]: You can use the `<fieldset>` html element for all of these form label and input pairs. This is better for accessibility 🙌  See more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset) in the MDN docs)

File: apps/src/signUpFlow/FinishStudentAccount.tsx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                name="isParent"
+              />
+            </div>
+            <BodyTwoText>{locale.i_am_a_parent_or_guardian()}</BodyTwoText>
+          </span>
+          {isParent && (
+            <>
+              <BodyThreeText className={style.studentQuestionLabel}>
+                <strong>{locale.parent_guardian_email()}</strong>
+              </BodyThreeText>
```
Comment (kelbyhawn) [inline apps/src/signUpFlow/FinishStudentAccount.tsx]: You can wrap all of the labels in a `<label>` element and connect it to their respective text field elements so when you click on the label it'll select the text field (also improves accessibility!). The `for` attribute connects to the `name` attribute on the input fields. 

```suggestion
            <label for="parentEmail">
              <BodyThreeText className={style.studentQuestionLabel}>
                <strong>{locale.parent_guardian_email()}</strong>
              </BodyThreeText>
            </label>
            ...
```

**Review by kelbyhawn — COMMENTED** [review]
This looks awesome! I left a couple comments for accessibility updates to the form that will enhance the UX too. 

@hannahbergam you may also want to take a look at this since there's more connect-y things happening!

File: apps/src/signUpFlow/FinishStudentAccount.tsx (hunk 1 above)
Comment (TurnerRiley) [reply apps/src/signUpFlow/FinishStudentAccount.tsx]: On it!

File: apps/src/signUpFlow/FinishStudentAccount.tsx (hunk 2 above)
Comment (TurnerRiley) [reply apps/src/signUpFlow/FinishStudentAccount.tsx]: On it!

**Review by kelbyhawn — APPROVED** [review]
So clean! 🤩

**Pull request merged**